core/src/main/java/org/stellar/anchor/sep24/Sep24Service.java

```diff
@@ -99,9 +97,12 @@ public InteractiveTransactionResponse withdraw(
       throw new SepValidationException("missing 'asset_code'");
     }
 
-    if (sourceAccount == null) {
-      info("missing 'account' field");
-      throw new SepValidationException("'account' is required");
+    if (isEmpty(sourceAccount)) {
+      sourceAccount = token.getAccount();
+      if (isEmpty(sourceAccount)) {
```

Comment:
I don't think we need this check, since SEP-10 is required and SEP-10 tokens always have an account? Same goes for the deposit endpoint.

Reply:
Removed both unnecessary changes.